### DISCUSSION TITLE
How to render file as HTML only, keeping images as separate files

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#3331] [Q&A]
In RStudio rendering makes not only HTML file but also [filename]_files - I don't want that. Making it `format.html.minimal: true;` makes some files disappear, but there is still `quarto-syntax-highlighting.css` and one more file. Making it `format.html.embed-resources: true` does the trick, but it affects images too and I want to keep them from file. I did not find fitting option, is there even?

**Comment by mcanouil** [comment]
You could generate the figures in a particular directory using `fig.path` (or `fig-path` in Quarto in theory) from `knitr`: `knitr.opts_chunk.fig-path: "directory_ending_with/"`. 🤔
